lib/device.js

```diff
@@ -7,6 +7,8 @@ var isIos = require('browser-detection/is-ios');
 var isIE9 = require('browser-detection/is-ie9');
 
 var CHROME_REGEX = /Chrome/;
+// Old Android Webviews used specific versions of Chrome with 0.0.0 as their version suffix
```

Comment:
Might be worth documenting this version behavior is for KitKat Chromium based WebViews

https://developer.chrome.com/multidevice/user-agent#webview_user_agent